prow/config/secret/agent.go

```diff
@@ -33,7 +33,8 @@ import (
 // Agent watches a path and automatically loads the secrets stored.
 type Agent struct {
 	sync.RWMutex
-	secretsMap map[string][]byte
+	secretsMap         map[string][]byte
+	secretRevisionsMap map[string]uint32
```

Comment:
Why not just int?

This seems overspecified. If you are going to specify it, when limit it to 32 bits?